client/src/main/webjar/program-form-dialog/program-form-dialog.controller.js

```diff
@@ -20,7 +20,7 @@
          * @returns promise.
          */
         function cancel() {
-            return $mdDialog.cancel(programFormDialog.requestedProgram);
+            return $mdDialog.cancel(programFormDialog.program);
```

Comment:
In this PR, I just blindly swapped out requestedProgram for program.

Issue #92 will fully integrate this, and actually make server calls.